### DISCUSSION TITLE
Import a Model and assign relations

### DISCUSSION THREAD

**Discussion by EtayZaslavsky** [django-import-export/django-import-export#1535] [Q&A]
Hi there!
I am creating a school system.
It has students, each classified to his school, grade, and class.
to make it easy to use, I wish to import new students from a spreadsheet file.
The school, grade and class objects are already created.

I want to assign each student to his class through his School name, Grade year, and ClassGroup number.

Couldn't make it with importing foreing key widger, neither with natural keys.
What is the right way to do it?

```
models.py

class School(models.Model):
    name = models.CharField(max_length=10, unique=True)

class Grade(models.Model):
    year = models.DecimalField(max_digits=4, decimal_places=0)
    school = models.ForeignKey(School, on_delete=models.CASCADE)

class ClassGroup(models.Model):
    number = models.DecimalField(max_digits=2, decimal_places=0)
    grade = models.ForeignKey(Grade, on_delete=models.CASCADE)

class Student(User):
    class_group = models.ForeignKey(ClassGroup)

```
![image](https://user-images.githubusercontent.com/105246259/216093894-abe0e1b2-11c4-414f-b33b-b6768af57ec7.png)

Thanks in advance!

**Comment by matthewhegarty** [comment]
Hi
I can't tell what your issue is here.  The best place to post is [Stackoverflow](https://stackoverflow.com/questions/tagged/django-import-export) (tag with 'django-import-export').  Please read '[how to ask](https://stackoverflow.com/help/how-to-ask)' first in order to get the best response to your question.

**Reply by EtayZaslavsky** [reply]
Thank you.
This solved it for me:[https://stackoverflow.com/questions/74647054/match-with-django-import-export-with-multiple-fields]( https://stackoverflow.com/questions/74647054/match-with-django-import-export-with-multiple-fields).
